docs/4_4_3_flexray.adoc

```diff
@@ -659,58 +659,81 @@ h|Kind h|Value h|Description
-
-.FMU parameter for the configuration of bus notifications within FlexRay.
+If a Network FMU is not supporting bus notifications at all, the configuration parameter shall be omitted.
+If a Network FMU supports bus notifications, the configuration parameter shall be defined with a default value set to `false` to allow the Network FMU to be run natively in each simulation scenario.
+If the value of the configuration parameter is set to `false`, the Network FMU must not wait for any response after a <<low-cut-flexray-transmit-operation, `Transmit`>> operation ("fire-and-forget").
+If the Network FMU shall be configured to handle responses in the form of <<low-cut-flexray-confirm-operation, `Confirm-`>> and <<low-cut-flexray-bus-error-operation, `Bus Error`>> operations, the <<low-cut-flexray-bus-notification-parameter, `BusNotifications`>> configuration parameter shall be set to `true`.
+A Network FMU must always be able to handle the case that the Bus Simulation does not provide <<low-cut-flexray-confirm-operation, `Confirm-`>> or <<low-cut-flexray-bus-error-operation, `Bus Error`>> operations.
+Since a Bus Simulation is allowed to provide bus notification regardless of the <<low-cut-flexray-bus-notification-parameter, `BusNotifications`>> configuration parameter, Network FMUs must silently ignore such notifications if the value of <<low-cut-flexray-bus-notification-parameter, `BusNotifications`>> is set to `false` or the parameter is omitted.
+
+.Configuration parameter to configure bus notifications within a FlexRay Bus Terminal of Network FMUs.
 [[figure-fmu-flexray-bus-notifications-parameter]]
 ----
-    org.fmi_standard.fmi_ls_bus.FlexRay_BusNotifications
-        Description:  "Specifies whether the respective Network FMU can rely on a Confirm,
-                      or Bus Error operation after sending a message."
-        Type:         Boolean
-        Causality:    structuralParameter
-        Variability:  fixed
-        Start:        "false"
+ MemberName:    BusNotifications
+ Description:   "Specifies whether the respective Network FMU can rely on a Confirm,
+                or Bus Error operation after sending a message."
+ Type:          Boolean
+ Causality:     parameter
+ Variability:   fixed
+ Start:         "false"
 ----
 
-This structural parameter is mandatory for Network FMUs.
-A Bus Simulation (FMU) does not require this structural parameter.
+FlexRay Bus Terminals of a Bus Simulation (FMU) might include this configuration parameter, by using a `calculatedParameter` or `constant output` to participate in an importer parameter propagation feature.
+
+.Configuration parameter to configure bus notifications within a FlexRay Bus Terminal of the Bus Simulation.
+[[figure-fmu-flexray-bus-notifications-parameter-in-bus-simulation]]
+----
+ MemberName:    BusNotifications
+ Description:   "Specifies whether the respective Bus Simulation provides bus notifications
```

Comment:
I think this list should only contain attributes that the ls requires to be set for the variable and for the member in the terminal. This should have the same spelling and capitalization like the attribute, e.g. `memberName`. The Description should not be listed like this, readers might even think that the description of the variable might have exactly this content.

Reply:
Good point! I will fix it with the next commit in a few minutes

Reply:
done